src/main/resources/hibernate.cfg.xml

```diff
@@ -0,0 +1,15 @@
+<hibernate-configuration>
+    <session-factory>
+        <property name = "hibernate.dialect">org.hibernate.dialect.MySQL5Dialect</property>
+        <property name = "connection.driver_class">com.mysql.cj.jdbc.Driver</property>
+        <property name = "connection.url">jdbc:mysql://localhost:3306/ticket_app?serverTimezone=UTC</property>
+        <property name = "connection.username">root</property>
+        <property name = "connection.password">root</property>
+        <property name="show_sql">true</property>
+        <property name="hbm2ddl.auto">create-drop</property>
+
+        <mapping class = "mate.academy.hibernate.relations.model.Actor"/>
+        <mapping class = "mate.academy.hibernate.relations.model.Country"/>
+        <mapping class = "mate.academy.hibernate.relations.model.Movie"/>
+    </session-factory>
+</hibernate-configuration>
```

Comment:
check if you have an empty line at the end of file

Reply:
everything is ok, it is shown on GitHub with red circle when a class does not end with an empty line